### ISSUE TITLE
Bronchial artery hypertrophy

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6527]
**Preferred term label:** Bronchial artery hypertrophy

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increase in the volume of bronchial artery wall due to the enlargement of its component cells.

**Parent term (use hpo.jax.org/app)**
Abnormality of bronchial artery (to be created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Pulmonary arterial hypertension associated with congenital heart disease

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added term
